pkg/controller/proposal/controller.go

```diff
@@ -456,7 +456,11 @@ func (r *Reconciler) reconcileCommit(ctx context.Context, proposal *configapi.Pr
 			if config.Values == nil {
 				config.Values = make(map[string]*configapi.PathValue)
 			}
+			// ToDo - remove before merging
+			log.Infof("Commit phase - Following paths are required to be updated:\n%v", changeValues)
```

Comment:
it might be better to make log.Debugf, and set the logging level to activate - then they could be left in permanently

Reply:
Log was completely removed - no need to keep it